In [1]:
import tushare as ts
print (ts.__version__)
ts.set_token('87b769832ea19e30b1f76de81c85a22c9bf5e88b7bf15678c2bb2e07')
pro = ts.pro_api()

1.2.62


In [8]:
df = pro.index_basic( ts_code="000905.SH")
df

ts_code   name market publisher category base_date  base_point list_date
0  000905.SH  中证500    SSE      中证公司     规模指数  20041231      1000.0  20070115

In [5]:
df=pro.stock_basic(market='SSE')


Empty DataFrame
Columns: [ts_code, symbol, name, area, industry, market, list_date]
Index: []

In [3]:
df = pro.daily(ts_code='603712.SH', start_date='20180301', end_date='20180331')

In [9]:
df = pro.index_daily(ts_code='000300.SH', start_date='20210301', end_date='20210304')

In [10]:
df

ts_code trade_date      close       open       high        low  \
0  000300.SH   20210304  5280.7058  5388.4785  5392.3725  5254.7771   
1  000300.SH   20210303  5452.2125  5327.3467  5452.3960  5316.4809   
2  000300.SH   20210302  5349.6301  5453.2847  5454.9687  5305.0545   
3  000300.SH   20210301  5418.7837  5389.2499  5421.5550  5345.9449   

   pre_close    change  pct_chg          vol       amount  
0  5452.2125 -171.5067  -3.1456  212775211.0  384382457.0  
1  5349.6301  102.5824   1.9176  194709054.0  337610313.6  
2  5418.7837  -69.1536  -1.2762  196511115.0  376444604.7  
3  5336.7609   82.0228   1.5369  177463523.0  358146546.0

In [12]:
df = pro.index_daily(ts_code='000905.SH', start_date='20210301', end_date='20210304')
df

ts_code trade_date      close       open       high        low  \
0  000905.SH   20210304  6401.9917  6482.7869  6485.1639  6371.7302   
1  000905.SH   20210303  6524.7513  6426.0369  6528.2217  6420.2068   
2  000905.SH   20210302  6433.3282  6514.0213  6514.0213  6386.4903   
3  000905.SH   20210301  6488.4388  6407.1642  6491.9367  6404.8885   

   pre_close    change  pct_chg          vol       amount  
0  6524.7513 -122.7596  -1.8814  194334616.0  192657769.9  
1  6433.3282   91.4231   1.4211  171017524.0  169667464.3  
2  6488.4388  -55.1106  -0.8494  165877069.0  174025453.4  
3  6364.3529  124.0859   1.9497  158608437.0  165792208.1

sh000300,2021-03-01,5389.2500,5421.5550,5345.9450,5418.7840,17746352000,358146539520.00,0,0,1

000300.SH	20210301	5418.7837	5389.2499	5421.5550	5345.9449	5336.7609	82.0228	1.5369	177463523.0	358146546.0

sh000300,2021-03-01,5389.2499,5421.5550,5345.9449,5418.7837,17746352300,358146546.0


000300.SH	20210302	5349.6301	5453.2847	5454.9687	5305.0545	5418.7837	-69.1536	-1.2762	196511115.0	376444604.7

sh000300,2021-03-02,5453.2847,5454.9687,5305.0545,5349.6301,19651111500,376444604700.00,0,0,1


000905.SH	20210302	6433.3282	6514.0213	6514.0213	6386.4903	6488.4388	-55.1106	-0.8494	165877069.0	174025453.4
sh000905,2021-03-02,6514.0213,6514.0213,6386.4903,6433.3282,16587706900,174025453400.00,0,0,1


sh000300,2021-03-02,5453.2847,5454.9687,5305.0545,5349.6301,19651111500,376444604700.00,0,0,1
sh000905,2021-03-02,6514.0213,6514.0213,6386.4903,6433.3282,16587706900,174025453400.00,0,0,1